src/libbson/src/bson/bson-json.c

```diff
@@ -1368,98 +1369,98 @@ _bson_json_read_map_key (bson_json_reader_t *reader, /* IN */
       _bson_json_save_map_key (bson, val, len);
    }
 
-   /* clang-format off */
    if (bson->read_state == BSON_JSON_IN_BSON_TYPE) {
-      if
```

Comment:
If you lead with an `if (false) {}`, then every subsequent HANDLE_OPTION can just be `else if`.

Reply:
I would strongly prefer to avoid hiding control flow statements within what is already an esoteric macro. The introduction of a dead code block feels in essence no different in spirit to an `#if 0`, which I would also like to avoid.